src/project.rs

```diff
@@ -507,7 +507,7 @@ impl FuzzProject {
             }
 
             eprintln!(
-                "Reproduce with:\n\n\tcargo fuzz run {target} {artifact}\n",
+                "Reproduce with:\n\n\tcargo fuzz run --debug {target} {artifact}\n",
```

Comment:
Let's leave these without any flags for simplicity, and handle adding flags for them in #228

Reply:
Done